tests/suite/layout/transform.typ

```diff
@@ -115,3 +115,53 @@ Hello #scaled[World]!
 #scale(x: auto, y: 50pt, reflow: true, cylinder)
 #scale(x: 100pt, y: auto, reflow: true, cylinder)
 #scale(x: 150%, y: auto, reflow: true, cylinder)
+
+--- transform-skew ---
```

Comment:
No need to increase the text and page size so much in all these tests. It bloats the size of the resulting images. It's intentional that everything is small and as a result a bit pixelated. I know that it makes them a bit harder to look at, but that's how it is (we want to keep the repository size from growing too much). For testing, you can always pass `--scale 2` or `--scale 4` to the test runner to render at a higher resolution.

Reply:
Okay, I will shrink the size of the reference images.